core/contracts/oracle/implementation/Store.sol

```diff
@@ -120,25 +121,28 @@ contract Store is StoreInterface, Withdrawable {
 
     /**
      * @notice Sets a new oracle fee per second.
-     * @param newOracleFee new fee per second charged to use the oracle.
+     * @param newOracleFeePerSecondPerPFC new fee per second charged to use the oracle.
      */
-    function setFixedOracleFeePerSecond(FixedPoint.Unsigned memory newOracleFee)
+    function setFixedOracleFeePerSecondPerPFC(FixedPoint.Unsigned memory newOracleFeePerSecondPerPFC)
         public
         onlyRoleHolder(uint(Roles.Owner))
     {
         // Oracle fees at or over 100% don't make sense.
-        require(newOracleFee.isLessThan(1));
-        fixedOracleFeePerSecond = newOracleFee;
-        emit NewFixedOracleFeePerSecond(newOracleFee);
+        require(newOracleFeePerSecondPerPFC.isLessThan(1));
+        fixedOracleFeePerSecondPerPFC = newOracleFeePerSecondPerPFC;
+        emit NewFixedOracleFeePerSecondPerPFC(newOracleFeePerSecondPerPFC);
     }
 
     /**
      * @notice Sets a new weekly delay fee.
-     * @param newWeeklyDelayFee fee escalation per week of late fee payment.
+     * @param newWeeklyDelayFeePerPFC fee escalation per week of late fee payment.
      */
-    function setWeeklyDelayFee(FixedPoint.Unsigned memory newWeeklyDelayFee) public onlyRoleHolder(uint(Roles.Owner)) {
-        weeklyDelayFee = newWeeklyDelayFee;
-        emit NewWeeklyDelayFee(newWeeklyDelayFee);
+    function setWeeklyDelayFeePerPFC(FixedPoint.Unsigned memory newWeeklyDelayFeePerPFC)
```

Comment:
This is actually becoming per second, too in [this PR](https://github.com/UMAprotocol/protocol/pull/1237). Do you want to change it to `setWeeklyDelayFeePerSecondPerPfc`? Or should I change the name in the aforementioned PR? Something else?